### PR TITLE
Bug/cutoff labels in bottom menu

### PR DESCRIPTION
**Pull request by badred123**
Fixed the bug where long strings that are passed as Building Names in the Bottom Menu are cut off because of the width of the label.

### CODE REVIEW

**Comment by AlainJobs** [comment]
Was this bug documented here on GitHub?

File: components/CurrentBuilding.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
```
Comment (AlainJobs) [inline components/CurrentBuilding.js]: Why is this file added though?

**Comment by csbduzi** [comment]
> Was this bug documented here on GitHub?

The bug was documented and he also linked the issue to the PR, if you look above

File: components/CurrentBuilding.js (hunk 1 above)
Comment (csbduzi) [reply components/CurrentBuilding.js]: He added it for the use of the useState and useEffect functions

**Comment by AlainJobs** [comment]
Sorry, I just saw! It's good to practise to add in the description of the PR as well. For futuer PRs

File: components/BottomMenu.js [hunk 2]
```diff
@@ -29,7 +29,13 @@ function BottomMenu ({ navigation }) {
 
     const getDestination = async () => {
         let searchItem = await AsyncStorage.getItem("toLocation");
-        setDestination(searchItem);
+        if(searchItem.length > 13){
+            var upadatedSearchItem = searchItem.substring(0,13) + "...";
+            setDestination(upadatedSearchItem);
+        }
```
Comment (csbduzi) [inline components/BottomMenu.js]: Nice fix for resolving the problem of long strings

File: components/BottomMenu.js [hunk 3]
```diff
@@ -38,6 +44,12 @@ function BottomMenu ({ navigation }) {
     //     setmapPressed(pressed);
     // };
 
+    const currentAltitude = async() => {
+        let altitude = await AsyncStorage.getItem('altitude');
+        console.log(altitude);
```
Comment (csbduzi) [inline components/BottomMenu.js]: You left a console.log() here
